client/verta/tests/test_organization.py

```diff
@@ -24,9 +26,75 @@ def test_create_msg():
 
 
 class TestOrganization:
-    @pytest.mark.skip("delete not implemented yet")
     def test_create(self, client):
         name = _utils.generate_default_name()
         org = client._create_organization(name)
         assert org
         assert org.id == client._get_organization(name).id
+
+        org.delete()
+
+    def test_create_same_name_diff_workspace(self, client, organization, in_tempdir, created_endpoints, created_registered_models, created_datasets):
+        # creating some entities:
+        project_name = _utils.generate_default_name()
+        exp_name = _utils.generate_default_name()
+        run_name = _utils.generate_default_name()
+        dataset_name = _utils.generate_default_name()
+        repository_name = _utils.generate_default_name()
+        model_name = _utils.generate_default_name()
+        version_name = _utils.generate_default_name()
+        endpoint_path = _utils.generate_default_name()
+
+        project = client.create_project(project_name)
+        exp = client.create_experiment(exp_name)
+        run = client.create_experiment_run(run_name)
+        repository = client.get_or_create_repository(name=repository_name)
+
+        dataset = client._create_dataset2(dataset_name)
+        created_datasets.append(dataset)
+        model = client.create_registered_model(name=model_name)
+        version = model.create_version(name=version_name)
+        created_registered_models.append(model)
+
+        endpoint = client.create_endpoint(path=endpoint_path)
+        created_endpoints.append(endpoint)
+
+        # Switching workspace name:
+        client_config = {
+            "workspace": organization.name
+        }
+
+        filepath = "verta_config.json"
+        with open(filepath, "w") as f:
+            json.dump(client_config, f)
+
+        client = Client()
```

Comment:
Could we use the `workspace` parameter on the client methods instead? I think I'd prefer making this test independent of the config file mechanisms.

Reply:
Fixed.